ItemList/Source/ItemList.DependencyInjection/Builder/UnityWebBootstrap.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Web.Http;
+using ItemList.Contracts.Bootstrap;
+using ItemList.DependencyInjection.Adapters;
+using Microsoft.Practices.Unity;
+
+namespace ItemList.DependencyInjection.Builder
+{
+    public class UnityWebBootstrap : IBootstrap
+    {
+        private readonly HttpConfiguration _config;
+        private readonly IEnumerable<IBootstrapper> _bootstrappers;
+
+        public UnityWebBootstrap(HttpConfiguration config, IEnumerable<IBootstrapper> bootstrappers)
+        {
+            _config = config;
+            _bootstrappers = bootstrappers;
+        }
+
+        public IBootstrap RegisterDependencies()
+        {
+            var container = new UnityContainer();
+
+            var unityAdapter = new UnityAdapter(container);
+            foreach (var instance in _bootstrappers)
+            {
+                instance.RegisterTypes(unityAdapter);
+            }
+
+            container.RegisterInstance<IBootstrap>(this);
```

Comment:
This line actually deserves a comment – why is it there and what does it mean (do not forget that all contracts should have an implementation and in a bigger system, you would actually want the system not to even start if there is a contract implementation missing).